docs/user_guide/discretisation/GeometricWidthDiscretiser.rst

```diff
@@ -0,0 +1,137 @@
+.. _increasing_width_discretiser:
+
+.. currentmodule:: feature_engine.discretisation
+
+GeometricWidthDiscretiser
+=========================
+
+The :class:`GeometricWidthDiscretiser()` divides continuous numerical variables into
+intervals of increasing width with equal increments. Note that the
+proportion of observations per interval may vary.
+
+Sizes of the intervals will follow geometric progression
+
+    .. math::
+        a_i+1 = a_i r^(n+i)
```

Comment:
This formula is not displaying correctly. I changed the () by {} so that r is at the power of n+1

However, this formula is not that of slide 48 from the presentation linked in the docs: https://www.slideshare.net/johnjreiser/classification-systems

So where is it coming from?

Reply:
I am still having trouble with the formula. r^(n+i) is the constant cw, so why n+i?

Reply:
Its here https://en.wikipedia.org/wiki/Geometric_progression